scene/gui/text_edit.cpp

```diff
@@ -3863,6 +3863,7 @@ void TextEdit::undo() {
 		}
 		caret_pos_dirty = true;
 	}
+	adjust_viewport_to_caret();
 	queue_redraw();
```

Comment:
`adjust_viewport_to_caret()` seems to already call `queue_redraw()`, so the other call is redundant.